tasks/runtime.yml

```diff
@@ -1,4 +1,12 @@
 ---
+- name: Fail if slurm_control_host or slurm_cluster_name or slurm_partitions are undefined
+  fail:
+    msg: "Undefined slurm_control_host or slurm_cluster_name or slurm_partitions."
+  when: 
+    slurm_control_host == none or
+    slurm_cluster_name == none or
+    slurm_partitions == 0
```

Comment:
slurm_partitions == 0? Isn't it a list?

`not slurm_partitions`

Reply:
Ah, there should be a `| length` in there somewhere

Reply:
Should be, for sure.

Reply:
Would be good provide a default empty list and perhaps fix the reference to cluster_login group here while you're at it: https://github.com/stackhpc/ansible-role-openhpc/blob/master/defaults/main.yml#L4